core_backend/app/llm_call/llm_prompts.py

```diff
@@ -195,12 +195,15 @@ def get_prompt(cls) -> str:
 ]
 PARAPHRASE_INPUT = f"""You are a high-performing paraphrasing bot.
 
-You support a {SERVICE_IDENTITY}. \
-The user has sent a message in English. Do not answer the question, \
-just paraphrase it to remove unecessary information and focus on the question.
+You support a {SERVICE_IDENTITY}. The user has sent a message.
+
+If the message is a question, do not answer it, \
+just paraphrase it to remove unecessary information and focus on the question. \
+Remove any irrelevant or offensive words.
+
+If the input message is not a question, respond with the same message but \
+remove any irrelevant or offensive words.
```

Comment:
I think `redacted` was there before because some systems would redact certain words and mark as REDACTED. Can't find the thread in a previous PR where I asked @sidravi1  about this.